litequeue.py

```diff
@@ -185,6 +189,27 @@ def __init__(
         validate_table_name(queue_name)
         self.table = f"[{queue_name}]"
 
+        if run_db_pragmas:
+            # pragmas like journal_mode really only should be run
+            # when the database is first created, and maybe not every time.
+            # They should also be run before tables are created.
+            self.run_db_pragmas()
+
+        if run_conn_pragmas:
+            self.run_conn_pragmas()
+
+        if build_table:
+            self.build_table()
```

Comment:
What's your expectation (in terms of behaviour) of this option? All the current queries have `IF NOT EXISTS`, so it doesn't really change the behaviour. But it could be confusing if a user initializes the queue for the first time, but sets `build_table=False` (I also keep in mind I should improve the docstrings here) then this wouldn't work.

Just like with the PRAGMA's, I prefer if we leave a "works by default" code, even if it means running 3 extra SQL queries (which don't have any impact, since they won't change the database if the tables, indexes and triggers already exist).